BracketTest.java

```diff
@@ -0,0 +1,22 @@
+import java.util.ArrayList;
+
+/**
+ * Created by Matthew on 5/1/2017.
+ */
+public class BracketTest {
+    public static void main(String[]args){
+        ArrayList<String> array = new ArrayList<String>();

```

Comment:
Be sure to initialize the bracket appropriately here - add 63 empty values, then 64 teams.